conda/common/path.py

```diff
@@ -113,12 +110,14 @@ def _process(x, y):
     return tuple('/'.join(leaf) for leaf in leaves)
 
 
-def explode_directories(child_directories, already_split=False):
+def explode_directories(child_directories: Iterable[tuple[str, ...]]):
```

Comment:
simplify signature (require already_split) since it's only called one way

Reply:
Unsure if we should deprecate this change:

```
def explode_directories(child_directories: Iterable[tuple[str, ...]], already_split=NULL):
    if already_split != NULL:
        warn(...)
```